core/src/main/java/de/danoeh/antennapod/core/util/Converter.java

```diff
@@ -30,9 +30,9 @@ public static String getDurationStringLong(int duration) {
 
     private static int[] millisecondsToHms(long duration) {
         int h = (int) (duration / HOURS_MIL);
-        long rest = duration - h * HOURS_MIL;
+        long rest = duration % HOURS_MIL;
         int m = (int) (rest / MINUTES_MIL);
-        rest -= m * MINUTES_MIL;
+        rest = duration % MINUTES_MIL;
```

Comment:
I don't think this change is necessary. It doesn't matter which way we do it, and this change just creates noise in the git history of that file.

Reply:
ok,I agree with you.